src/vs/base/parts/quickinput/common/quickInput.ts

```diff
@@ -6,14 +6,24 @@
 import { ResolvedKeybinding } from 'vs/base/common/keyCodes';
 import { URI } from 'vs/base/common/uri';
 import { Event } from 'vs/base/common/event';
+import { IDisposable } from 'vs/base/common/lifecycle';
+import { IMatch } from 'vs/base/common/filters';
+
+export interface IQuickPickItemHighlights {
+	label?: IMatch[];
+	description?: IMatch[];
+	detail?: IMatch[];
+}
 
 export interface IQuickPickItem {
 	type?: 'item';
 	id?: string;
 	label: string;
+	ariaLabel?: string;
```

Comment:
@chrmarti fyi this is for being able to define an ARIA label that is different from the pick item label